README.md

```diff
@@ -23,9 +23,11 @@ npm install @stripe/stripe-js
 ### `loadStripe`
 
 This function returns a `Promise` that resolves with a newly created `Stripe`
-object once Stripe.js has loaded. If necessary, it will load Stripe.js for you
-by inserting the Stripe.js script tag. If you call `loadStripe` in a server
-environment it will resolve to `null`.
+object once Stripe.js has loaded. It takes the same parameters you normally pass
```

Comment:
(nit)
```suggestion
object once Stripe.js has loaded. It takes the same parameters passed
```
Since using the loader might be the only / normal way for someone to instantiate a `Stripe` object.